card/lib/card/model/save_helper.rb

```diff
@@ -83,6 +83,29 @@ def ensure_card! name_or_args, content_or_args=nil
         end
       end
 
+      def ensure_trait name, codename, args
+        ensure_card name, codename: codename
+        args.each do |k, v|
+          send "ensure_trait_#{k}", name, v
```

Comment:
That module is only used for migrations and testing but sure, more sensible errors messages never hurt